akinet/tcp_parser.go

```diff
@@ -14,6 +14,14 @@ const (
 	NeedMoreData
 )
 
+const (
+	CONNECTION_TYPE_HTTP_CLIENT   = "HTTP_CLIENT"
+	CONNECTION_TYPE_HTTP_SERVER   = "HTTP_SERVER"
+	CONNECTION_TYPE_TLS_CLIENT    = "TLS_CLIENT"
+	CONNECTION_TYPE_TLS_SERVER    = "TLS_SERVER"
+	CONNECTION_TYPE_HTTP2_PREFACE = "HTTP2_PREFACE"
+)
```

Comment:
Is it worth having these defined for each protocol type, or should we have only client and server?  (Or unknown?)